<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>671</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="mainScroll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-180</y>
         <width>805</width>
         <height>1078</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="0">
         <layout class="QGridLayout" name="buttonGrid" rowstretch="0,0,0,0,0,0,0">
          <item row="5" column="0">
           <widget class="QPushButton" name="printSurgeryReport">
            <property name="text">
             <string>Print Surgery Report</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="printChartNotesReport">
            <property name="text">
             <string>Print Chart Notes Report</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="addImplantYes">
            <property name="text">
             <string>Yes</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="Line" name="line">
            <property name="midLineWidth">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="addImplantNo">
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Add Another Implant</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTabWidget" name="createPages">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page1">
           <attribute name="title">
            <string>Implant 1</string>
           </attribute>
           <attribute name="toolTip">
            <string>Implant 1</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4" columnstretch="0" rowminimumheight="0,0">
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="implantLayout">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="topGrid" rowstretch="0,0,0,0,0,0,0" columnstretch="0,2,1">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <item row="6" column="1">
           <widget class="QDateEdit" name="restoreDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateEdit" name="date">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Chart Number</string>
            </property>
            <property name="buddy">
             <cstring>chartNumber</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Doctor</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDateEdit" name="uncoverDate">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Date</string>
            </property>
            <property name="buddy">
             <cstring>date</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QCheckBox" name="singleStage">
            <property name="text">
             <string>Single Stage</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="chartNumber"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="patientName"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Uncover Date</string>
            </property>
            <property name="buddy">
             <cstring>uncoverDate</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Restore Date</string>
            </property>
            <property name="buddy">
             <cstring>restoreDate</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Patient Name</string>
            </property>
            <property name="buddy">
             <cstring>patientName</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QVBoxLayout" name="doctorBox"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="reportGrid">
          <item row="18" column="0">
           <widget class="QGroupBox" name="implantRestoreBox">
            <property name="title">
             <string>Implant Restore</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <property name="verticalSpacing">
              <number>11</number>
             </property>
             <item row="3" column="0">
              <widget class="QRadioButton" name="restoreBtn4">
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QRadioButton" name="restoreBtn6">
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="restoreOpt1">
               <property name="text">
                <string>Implant can be restored after &lt;Implant restore date&gt;.  For posterior implants with a good path of draw we recommend screw retained restorations.  </string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>restoreBtn1</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="restoreOpt3">
               <property name="text">
                <string>Implant can be restored after &lt;Implant restore date&gt;.  A screw retained restoration should be possible using the Zimmer angled screw channel (Certain BellaTek Express &amp; Flex abutments).  This is, of course, at the discretion of the restoring doctor.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>restoreBtn3</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="restoreOpt2">
               <property name="text">
                <string>Implant can be restored after &lt;Implant restore date&gt;.  A screw retained restoration should be possible using the NobelBiocare Angled Screw Channel.  This is, of course, at the discretion of the restoring doctor</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>restoreBtn2</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="restoreBtn3">
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="restoreBtn2">
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="restoreBtn1">
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QRadioButton" name="restoreBtn5">
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="restoreOpt4">
               <property name="text">
                <string>Implant can be uncovered after &lt;implant uncovery date&gt; and restored after &lt;Implant restore date&gt;.  For posterior implants with a good path of draw we recommend screw retained restorations.  </string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>restoreBtn4</cstring>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLabel" name="restoreOpt5">
               <property name="text">
                <string>Implant can be uncovered after &lt;implant uncovery date&gt; and restored after &lt;Implant restore date&gt;.  A screw retained restoration should be possible using the NobelBiocare Angled Screw Channel.  This is, of course, at the discretion of the restoring doctor</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>restoreBtn5</cstring>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="restoreOpt6">
               <property name="text">
                <string>Implant can be uncovered after &lt;implant uncovery date&gt;  restored after &lt;Implant restore date&gt;.  A screw retained restoration should be possible using the Zimmer angled screw channel (Certain BellaTek Express &amp; Flex abutments).  This is, of course, at the discretion of the restoring doctor.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>restoreBtn6</cstring>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="bottomGrid">
          <item row="0" column="0">
           <widget class="QGroupBox" name="anestheticBox">
            <property name="title">
             <string>Anesthetic</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QRadioButton" name="radioButton_46">
               <property name="text">
                <string>2% Xylocaine (1:100,000 epi)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_54">
               <property name="text">
                <string>2% Xylocaine (1:50,000 epi)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_49">
               <property name="text">
                <string>3% Mepivicaine</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_48">
               <property name="text">
                <string>2% Septicaine</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_47">
               <property name="text">
                <string>2% Marcaine</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="prescriptionsBox">
            <property name="title">
             <string>Prescriptions</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QCheckBox" name="checkBox_3">
               <property name="text">
                <string>Amoxicillin 500mg; 30 caps; Take 1 cap tid til gone</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_9">
               <property name="text">
                <string>PenVK 500mg; 30 caps; Take 1 cap tid til gone</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_8">
               <property name="text">
                <string>Cephelexin 500mg; 32 caps; Take 1 cap qid til gone</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_7">
               <property name="text">
                <string>Clindamycin 300mg; 30 caps; Take 1 cap tid til gone</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_6">
               <property name="text">
                <string>Ibuprofen 600mg; 24 tabs; Take 1 tab q6h</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_5">
               <property name="text">
                <string>Norco-5; 24 tabs; Take 1 tab between doses of Ibuprofen prn pain</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_4">
               <property name="text">
                <string>Norco-5; 24 tabs; Take 1 tab q4-6h prn pain</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="toleranceBox">
            <property name="title">
             <string>Patient Tolerance</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="radioButton_50">
               <property name="text">
                <string>Excellent</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_53">
               <property name="text">
                <string>Good</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_52">
               <property name="text">
                <string>Fair</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_51">
               <property name="text">
                <string>Poor</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="addXray">
            <property name="text">
             <string>Upload X-Ray</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="xrayPath">
            <property name="text">
             <string>File Path: </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>patientName</tabstop>
  <tabstop>chartNumber</tabstop>
  <tabstop>date</tabstop>
  <tabstop>uncoverDate</tabstop>
  <tabstop>singleStage</tabstop>
  <tabstop>restoreDate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
